plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -174,52 +164,35 @@ public void testCreateTableUnsupportedType(String createType)
     @Test
     public void testCreateTableWithRowTypeWithoutField()
     {
+        // static name to make sure test fails if table exists (since we don't expect table to get created)
```

Comment:
it's still ok to use random table name here

eg, should the test succeed once (on some branch/PR), it could start failing with "table already exists".